src/app/project-dialog/project-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="col-md-12 col-sm-12 col-xs-12 createProjectBlock">
+    <div class="col-md-12 col-sm-12 col-xs-12">
+        <h4 class="projectname">Project Name</h4>
+        <form role="form" class="search-pf has-button">
+            <div class="form-group has-clear">
+                <div class="search-pf-input-group">
+                    <input id="projectname" type="search" class="form-control" placeholder="Enter a unique name for your project">
+                </div>
+            </div>
+            <div class="form-group">
+                <button class="btn btn-default" type="button"><span class="caret pull-right icon-caret"></span></button>
+            </div>
+        </form>
+        <div class="form-group has-error">
+            <div class="search-pf-input-group">
+                <input id="projectname" type="search" class="form-control" placeholder="BalloonPopGame">
+                <span class="help-block">This project name already exists. Please enter another name.</span>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-12 col-sm-12 col-xs-12">
+        <h4 class="projectname">Development Process</h4>
+        <form role="form" class="search-pf has-button">
+            <div class="form-group has-clear">
+                <div class="search-pf-input-group">
+                    <input id="projectname" type="search" class="form-control" placeholder="Choose a Development process for this project">
+                </div>
+            </div>
+            <div class="form-group">
+                <button class="btn btn-default" type="button"><span class="caret pull-right icon-caret"></span></button>
+            </div>
+        </form>
+        <p class="pull-left">Agile</p>
+    </div>
+
+    <div class="col-md-12 col-sm-12 col-xs-12 projectInputBlock">
+        <h4 class="projectname">What type of project would you like to create</h4>
+        <div class="col-md-12 col-sm-12 col-xs-12">
+            <input type="radio" class="pull-left" id="publicProject" checked/>
```

Comment:
Add name="projectView" to the input element. This will select the radio button one at a time.